.github/workflows/continuous-delivery.yaml

```diff
@@ -39,12 +39,19 @@ jobs:
           commit-assets: |
             ./dist
           node-module: true
+          release-rules:
+            '[ { "release": "patch", "type": "build" }, { "release": "patch",
+            "type": "chore", "scope": "deps" }, { "release": "patch", "type":
+            "chore", "scope": "deps-dev", "message": "*typescript*" }, {
+            "release": "patch", "type": "ci" }, { "release": "patch", "type":
+            "docs" }, { "release": "patch", "type": "improvement" }, {
+            "release": "patch", "type": "refactor" }, ]'
```

Comment:
Can we somehow format it? I can't read it without copy-pasting to an editor.

Reply:
I did, however, it was overridden by Prettier :(

Reply:
Btw., it's not a valid JSON!

Reply:
It's a YAML file, YAML has multiple ways to format strings and you'll definitely find one that you can format yourself, e.g. `: |` syntax.